packages/@aws-cdk/aws-cognito/lib/user-pool-email.ts

```diff
@@ -0,0 +1,267 @@
+import { Stack, Token } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { toASCII as punycodeEncode } from 'punycode/';
+
+/**
+  * The valid Amazon SES configuration regions
+  */
+const REGIONS = ['us-east-1', 'us-west-2', 'eu-west-1'];
+
+/**
+ * Configuration for Cognito sending emails via Amazon SES
+ */
+export interface SESOptions {
```

Comment:
Since this module contains both user pool and identity pool, perhaps rename this as `UserPoolSESOptions`.

Same for other names as well.

Reply:
Updated all the exported names.